Tasks/AzureRmWebAppDeploymentV3/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -78,7 +78,7 @@
   "loc.input.label.RemoveAdditionalFilesFlag": "Remove additional files at destination",
   "loc.input.help.RemoveAdditionalFilesFlag": "Select the option to delete files on the Azure App Service that have no matching files in the App Service package or folder.<br /><br />Note: This will also remove all files related to any extension installed on this Azure App Service. To prevent this, select 'Exclude files from App_Data folder' checkbox. ",
   "loc.input.label.ExcludeFilesFromAppDataFlag": "Exclude files from the App_Data folder",
-  "loc.input.help.ExcludeFilesFromAppDataFlag": "Select the option to prevent files in the App_Data folder from being deployed to the Azure App Service.",
+  "loc.input.help.ExcludeFilesFromAppDataFlag": "Select the option to prevent files in the App_Data folder from being deleted from the Azure App Service.",
```

Comment:
Since this flag excludes App-Data folder from deployment, saving new files to be deployed and existing files from depletion. Can we have both in the text? "from being deployed to/ deleted from the Azure App Service."

Reply:
Also can you update the task patch version for both 3.* and 4.*?

Reply:
Fixed both comments. How about the others languages for the resources? Are they generated somewhere?